src/Catalog/Helpers/DeletionAuditEntry.cs

```diff
@@ -114,14 +114,29 @@ private void InitValues()
         /// <summary>
         /// Fetches <see cref="DeletionAuditEntry"/>s.
         /// </summary>
+        /// <param name="auditingStorageFactory"></param>
+        /// <param name="cancellationToken"></param>
+        /// <param name="package"></param>
+        /// <param name="minTime"></param>
+        /// <param name="maxTime"></param>
+        /// <param name="logger"></param>
+        /// <returns></returns>
```

Comment:
I swear I added documentation here...I think VS must have crashed and lost it lol